app/services/slack_notification_service.rb

```diff
@@ -13,7 +13,7 @@ class SlackNotificationService
     'Python': 'python',
     'React': 'react',
     'React-Native': 'react-native',
-    'Flutter': 'flutter',
+    'Dart': 'dart',
```

Comment:
mmm I don't think this is a good change, even when Dart is the language we are working with the FW Flutter that is known by everyone. I would add a conditional to keep the same Flutter channel

Reply:
@JuanfraM  Done. We had a discussion about that and resolved to take this approach